build-binutils.py

```diff
@@ -0,0 +1,172 @@
+                        target prefix if it is not for the host architecture.
+                        """,
+                        default="all",
+                        nargs="+")
+    return parser.parse_args()
+
+
+def create_targets(targets):
+    targets_dict = {
+        "arm": "arm-linux-gnueabi",
+        "aarch64": "aarch64-linux-gnu",
+        "powerpc64le": "powerpc64le-linux-gnu",
+        "powerpc": "powerpc-linux-gnu",
+        "x86_64": "x86_64-linux-gnu"
+    }
+    targets_list = []
+
+    if ''.join(targets) == "all":
+        for key in targets_dict:
+            targets_list.append(targets_dict[key])
+    else:
+        for target in targets:
+            if target == "host":
+                key = host_to_key()
+            else:
+                key = target_to_arch(target)
+            targets_list.append(targets_dict[key])
+
+    return targets_list
+
+
+def cleanup(build_folder):
+    if build_folder.is_dir():
+        shutil.rmtree(build_folder.as_posix())
+    build_folder.mkdir(parents=True, exist_ok=True)
+
+
+def invoke_configure(build_folder, install_folder, root, target):
+    configure = [
+        root.joinpath(utils.current_binutils(), "configure").as_posix(),
+        '--prefix=' + install_folder.as_posix(),
+        '--enable-deterministic-archives', '--enable-gold',
+        '--enable-ld=default', '--enable-plugins', '--quiet',
+        'CFLAGS=-O2 -march=native -mtune=native',
+        'CXXFLAGS=-O2 -march=native -mtune=native'
+    ]
+    if "arm" in target or "aarch64" in target:
+        configure += [
+            '--disable-multilib', '--disable-nls',
+            '--program-prefix=' + target + '-', '--target=' + target,
+            '--with-gnu-as', '--with-gnu-ld',
+            '--with-sysroot=' + install_folder.joinpath(target).as_posix()
+        ]
+    elif "powerpc" in target:
+        configure += [
+            '--enable-lto', '--enable-relro', '--enable-shared',
+            '--enable-threads', '--disable-gdb', '--disable-sim',
+            '--disable-werror', '--program-prefix=' + target + '-',
+            '--target=' + target, '--with-pic', '--with-system-zlib'
+        ]
+    elif "x86_64" in target:
+        configure += [
+            '--enable-lto', '--enable-relro', '--enable-shared',
+            '--enable-targets=x86_64-pep', '--enable-threads', '--disable-gdb',
+            '--disable-werror', '--with-pic', '--with-system-zlib'
+        ]
+    # If the current machine is not the target, add the prefix to indicate
+    # that it is a cross compiler
+    if not host_is_target(target):
+        configure += ['--program-prefix=' + target + '-', '--target=' + target]
+    utils.header("Building " + target + " binutils")
+    subprocess.run(configure, check=True, cwd=build_folder.as_posix())
+
+
+def invoke_make(build_folder, install_folder, target):
+    if host_is_target(target):
+        subprocess.run(['make', '-s', 'configure-host', 'V=0'],
```

Comment:
`V=0` is the default for `make`, right?

Reply:
Yes, it should be, will remove.

Reply:
https://github.com/MSF-Jarvis/tc-build/commit/4f2baa816e9d1bc66d6c4a04c80c59b921ac9e21

Reply:
I actually think the default for binutils might be V=1 because the build is much noisier without V=0 so I think we'll keep it.